pkg/controllers/vscr/scrutinizepod_reconciler.go

```diff
@@ -92,7 +92,7 @@ func (s *ScrutinizePodReconciler) Reconcile(ctx context.Context, _ *ctrl.Request
 // to the vcluster scrutinize CLI
 func (s *ScrutinizePodReconciler) collectInfoFromVdb(ctx context.Context) (ctrl.Result, error) {
 	finder := iter.MakeSubclusterFinder(s.VRec.Client, s.Vdb)
-	pods, err := finder.FindPods(ctx, iter.FindExisting)
+	pods, err := finder.FindPods(ctx, iter.FindExisting, v1.MainCluster)
```

Comment:
I guess we can't ever collect scrutinize inside a sandbox. Should have a jira for this?

Reply:
You mean to collect scrutinize inside a sandbox? Yes we can I will open a jira for that.

Reply:
Yes, exactly.